Resources/Prototypes/Species/vox.yml

```
@@ -6,7 +6,7 @@
   sprites: MobVoxSprites
   markingLimits: MobVoxMarkingLimits
   dollPrototype: MobVoxDummy
-  skinColoration: VoxFeathers
+  skinColoration: Hues # imp
```

```suggestion
  skinColoration: Hues # imp VoxFeathers<Hues
```
Note original value for future maintenance work.